frontend/src/components/panels/project_list_panel/ProjectCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const ProjectCard = () => (
```

Comment:
It would be more clear as to the props that a component uses by declaring them as follows

```javascript
const ProjectCard = ({ iconUrl, myOtherProp }) => (
    <div>{iconUrl}</div>
);
```